legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -55,6 +56,11 @@ def get_businesses(identifier: str):
     if recent_filing_json:
         business_json['submitter'] = recent_filing_json['filing']['header']['submitter']
         business_json['lastModified'] = recent_filing_json['filing']['header']['date']
+
+    allowed_filings = str(request.args.get('allowed_filings', None)).lower() == 'true'
```

Comment:
I think spinal-case (aka kebab-case) is the recommendation for URL parameters, but check if we already use a specific convention.